### PR TITLE
Add "F" to EvaluationType

### PR DESCRIPTION
**Pull request by buckley-w-david**
This PR adds "F" to EvaluationType, as while it is not still in use, it was in use for some time in the historical data that is in scope.

### CODE REVIEW

File: nrcan_etl/src/energuide/embedded/evaluation_type.py [hunk 1]
```diff
@@ -5,12 +5,15 @@
 class EvaluationType(enum.Enum):
     PRE_RETROFIT = 'D'
     POST_RETROFIT = 'E'
+    INSENTIVE_PROGRAM = 'F'
```
Comment (evadb) [inline nrcan_etl/src/energuide/embedded/evaluation_type.py]: Does "INSENTIVE" stand for something in particular? Is it possibly INCENTIVE?

Comment (buckley-w-david) [reply nrcan_etl/src/energuide/embedded/evaluation_type.py]: Good catch, spelling is my weak point

File: nrcan_etl/src/energuide/embedded/evaluation_type.py [hunk 2]
```diff
@@ -5,12 +5,15 @@
...
 
     @classmethod
     def from_code(cls, code: str) -> 'EvaluationType':
         if code == cls.PRE_RETROFIT.value:
             return EvaluationType.PRE_RETROFIT
         elif code == cls.POST_RETROFIT.value:
             return EvaluationType.POST_RETROFIT
+        elif code == cls.INSENTIVE_PROGRAM:
```
Comment (evadb) [inline nrcan_etl/src/energuide/embedded/evaluation_type.py]: Should they all be changed to INCENTIVE?

File: nrcan_etl/src/energuide/embedded/evaluation_type.py (hunk 1 above)
Comment (evadb) [reply nrcan_etl/src/energuide/embedded/evaluation_type.py]: I became a programmer to avoid spelling tbh

**Pull request merged**